src/app/fdctl/configure/xdp.c

```diff
@@ -86,7 +86,7 @@ init( config_t * const config ) {
     config->tiles.quic.regular_transaction_listen_port,
     config->tiles.quic.quic_transaction_listen_port,
     config->tiles.shred.shred_listen_port,
-    config->tiles.gossip.gossip_listen_port,
+    config->gossip.port,
```

Comment:
This change (already) breaks 1.5 since you are going to intercept gossip traffic from reaching the agave code, even when running 1.5 topology.